fdk/http/event_handler.py

```diff
@@ -16,25 +16,30 @@
 import logging
 
 from fdk import constants
-from fdk import customer_code
-from fdk import runner
 
 logger = logging.getLogger(__name__)
 
 
-def event_handle(handle_code: customer_code.Function):
+def event_handle(handle_code):
     """
     Performs HTTP request-response procedure
     :param handle_code: customer's code
     :type handle_code: fdk.customer_code.Function
     :return: None
     """
     async def pure_handler(request):
+        from fdk import runner
         logger.info("in pure_handler")
         func_response = await runner.handle_request(
             handle_code, constants.HTTPSTREAM,
             headers=dict(request.headers), data=io.BytesIO(request.body))
         logger.info("request execution completed")
-        return func_response
+
+        from async_http import response
```

Comment:
same, import response only you need that, import are cachable, so, only 1st import is heavy, the rest are not.